src/frontend/Sider/Behandling/VedtakOgBeregning/Barnetilsyn/InnvilgeVedtak/VedtaksperiodeRad.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+
+import { TrashIcon } from '@navikt/aksel-icons';
+import { Button } from '@navikt/ds-react';
+
+import { useBehandling } from '../../../../../context/BehandlingContext';
+import { FormErrors } from '../../../../../hooks/felles/useFormState';
+import { useRevurderingAvPerioder } from '../../../../../hooks/useRevurderingAvPerioder';
+import DateInputMedLeservisning from '../../../../../komponenter/Skjema/DateInputMedLeservisning';
+import SelectMedOptions from '../../../../../komponenter/Skjema/SelectMedOptions';
+import { BehandlingType } from '../../../../../typer/behandling/behandlingType';
+import { VedtaksperiodeTilsynBarn } from '../../../../../typer/vedtak/vedtakTilsynBarn';
+import {
+    aktivitetTypeTilTekst,
+    valgbareAktivitetTyperForStønadsperiode,
+} from '../../../Inngangsvilkår/Aktivitet/utilsAktivitet';
+import { StatusTag } from '../../../Inngangsvilkår/Stønadsperioder/StatusTag';
+import {
+    målgruppeTypeOptionsForStønadsperiode,
+    målgruppeTypeTilTekst,
+} from '../../../Inngangsvilkår/typer/vilkårperiode/målgruppe';
+
+interface Props {
+    vedtaksperiode: VedtaksperiodeTilsynBarn;
+    erLesevisning: boolean;
```

Comment:
For at vi skal validere vedtaksperioder riktig så burde vi nog håndtere vedtaksperioder på lik måte som her https://github.com/navikt/tilleggsstonader-sak-frontend/pull/697
Men den endringen kan gjøres etterpå

Reply:
Jeps! Legger det inn snere 👍